# HG changeset patch
# User Mike Pavone <pavone@retrodev.com>
# Date 1594527392 25200
#      Sat Jul 11 21:16:32 2020 -0700
# Node ID e45a317802bdcdbf259d9daf0620059a93a15251
# Parent  7a6835c02db0cd78fcbf0d315a4705919b922c0f
Fix broken enum definitions that cause multiple definition errors when building with -fno-common which is now the default in GCC 10

diff -r 7a6835c02db0 -r e45a317802bd gen_x86.c
--- a/gen_x86.c	Sat Jul 11 21:04:16 2020 -0700
+++ b/gen_x86.c	Sat Jul 11 21:16:32 2020 -0700
@@ -130,7 +130,7 @@
 	X86_R13,
 	X86_R14,
 	X86_R15
-} x86_regs_enc;
+};
 
 char * x86_reg_names[] = {
 #ifdef X86_64
diff -r 7a6835c02db0 -r e45a317802bd gen_x86.h
--- a/gen_x86.h	Sat Jul 11 21:04:16 2020 -0700
+++ b/gen_x86.h	Sat Jul 11 21:16:32 2020 -0700
@@ -30,7 +30,7 @@
 	R13,
 	R14,
 	R15
-} x86_regs;
+};
 
 enum {
 	CC_O = 0,
@@ -51,14 +51,14 @@
 	CC_GE,
 	CC_LE,
 	CC_G
-} x86_cc;
+};
 
 enum {
 	SZ_B = 0,
 	SZ_W,
 	SZ_D,
 	SZ_Q
-} x86_size;
+};
 
 #ifdef X86_64
 #define SZ_PTR SZ_Q
@@ -85,7 +85,7 @@
 	MODE_REG_DIRECT = 0xC0,
 //"phony" mode
 	MODE_IMMED = 0xFF
-} x86_modes;
+};
 
 void rol_ir(code_info *code, uint8_t val, uint8_t dst, uint8_t size);
 void ror_ir(code_info *code, uint8_t val, uint8_t dst, uint8_t size);
diff -r 7a6835c02db0 -r e45a317802bd vdp.h
--- a/vdp.h	Sat Jul 11 21:04:16 2020 -0700
+++ b/vdp.h	Sat Jul 11 21:16:32 2020 -0700
@@ -92,7 +92,7 @@
 	REG_DMASRC_L,
 	REG_DMASRC_M,
 	REG_DMASRC_H
-} vdp_regs;
+};
 
 //Mode reg 1
 #define BIT_VSCRL_LOCK 0x80
